master/server.go

```diff
@@ -166,6 +198,31 @@ func (s *Server) Start(ctx context.Context) (err error) {
 	if test.GlobalTestFlag {
 		return s.startForTest(ctx)
 	}
+
+	err = s.startGrpcSrv()
+	if err != nil {
+		return
+	}
+
+	// start leader election
+	// TODO: Consider election. And Notify workers when leader changes.
+	// s.election, err = election.NewElection(ctx, )
+
+	// rebuild states from existing meta if needed
+	err = s.resetExecutor(ctx)
+	if err != nil {
+		return err
+	}
+
+	// start background managers
+	s.executorManager.Start(ctx)
+	s.jobManager.Start(ctx)
```

Comment:
the failover of jobmanager will be  implemented in the JobManager Start()  method ?

Reply:
Will be implemented in another PR.